.rubocop_todo.yml

```diff
@@ -1,6 +1,6 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2022-08-08 14:26:32 UTC using RuboCop version 1.33.0.
+# on 2023-12-27 11:12:52 UTC using RuboCop version 1.59.0.
```

Comment:
Seems not completely relevant, but, I guess, acceptable.

Reply:
Timestamp maybe not, I agree, but the RuboCop version can be useful 👍

Reply:
Yeah, I'm generally talking more about RuboCop itself… the PR is about minimal Ruby version, and there are RuboCop updates. But it's up to you, I'm not against it.

Reply:
Ah right, good point. I've limited changes to those necessary after increasing the `TargetRubyVersion` rubocop config from 2.6 to 3.0. This change was required because Rubocop needs that to be the same as the Gemspec's `required_ruby_version`. After that some new offences popped up